src/themes/index.js

```diff
@@ -9,6 +9,7 @@ import { barebonesTheme } from './barebones.js'
 // import  { materializeTheme } from  './materialize'
 import { spectreTheme } from './spectre.js'
 import { tailwindTheme } from './tailwind.js'
+import { silvereditTheme } from './silveredit.js'
```

Comment:
we can add a theme without having our fork for json-editor

Reply:
How? By just maintaining  a local version of json-editor somewhere and putting build on s3?